azure-pipelines.yml

```diff
@@ -7,8 +7,9 @@ trigger:
 pr: none
 
 variables:
-  buildConfiguration: 'release'
-  buildPlatform: 'anycpu'
+  - group: Prod Management Resources
```

Comment:
Is this variable group needed for the build?

Reply:
Its needed for the dependency check variables